zebra-chain/src/block/tests/generate.rs

```diff
@@ -256,8 +244,8 @@ fn single_transaction_block_many_outputs(oversized: bool) -> Block {
     // A block header
     let (block_header, block_header_bytes) = block_header();
 
-    // A LockTime
-    let (lock_time, lock_time_bytes) = lock_time();
+    let lock_time = LockTime::Time(DateTime::from_timestamp(61, 0).unwrap());
```

Comment:
Optional/Nitpick: Do we still want to use the updated method if the deprecated one has been restored?

Reply:
The removed `lock_time` fn was ours. It used the `from_timestamp_opt` fn, which is deprecated even in the newest version of `chrono`, and the recommended one is `from_timestamp`, which we're using. The deprecation here means that there's now only a warning when using `from_timestampt_opt`.

The other deprecated fns in the previous version of `chrono` were actually removed, and Zebra wouldn't compile. The newer version has them back without deprecation warnings.